### PR TITLE
Edge/10.17 rn update

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-17-0/whatsnew-10-17-0/).
+
+## Cumulocity IoT Edge on Kubernetes - Public Preview
+
+Cumulocity IoT Edge can now be deployed and managed on a single node Kubernetes cluster using Cumulocity IoT Edge Kubernetes Operator (Edge Operator).
+
+Edge on Kubernetes managed by the Edge Operator enables you with:
+
+- **Unified Management**: Manage Edge through an Edge Custom Resource (CR) in consistent with standard Kubernetes resources. This approach provides a unified and familiar interface for administrators and operators.
+- **Desired State Management**: Specify the desired state of Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:33]:
```suggestion
- **Desired State Management**: Specify the desired state of Cumulocity IoT Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 
+- **Automation**: Edge CR empowers you to automate various operational aspects of Edge like scaling, upgrades, and configuration.
+- **Distribution**: Distribute and replicate the Edge deployments using Edge CR. Simplifies the deployment and accelerates the adoption of Cumulocity IoT platform across diverse environments.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:36]:
```suggestion
- **Distribution**: Distribute and replicate Cumulocity IoT Edge deployments using the Edge CR. This simplifies the deployment and accelerates the adoption of the Cumulocity IoT platform across different environments.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Cumulocity IoT Edge on Kubernetes - Public Preview
+
+Cumulocity IoT Edge can now be deployed and managed on a single node Kubernetes cluster using Cumulocity IoT Edge Kubernetes Operator (Edge Operator).
+
+Edge on Kubernetes managed by the Edge Operator enables you with:
+
+- **Unified Management**: Manage Edge through an Edge Custom Resource (CR) in consistent with standard Kubernetes resources. This approach provides a unified and familiar interface for administrators and operators.
+- **Desired State Management**: Specify the desired state of Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
+- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 
+- **Automation**: Edge CR empowers you to automate various operational aspects of Edge like scaling, upgrades, and configuration.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:35]:
```suggestion
- **Automation**: Edge CR empowers you to automate various operational aspects of Cumulocity IoT Edge like scaling, upgrades, and configuration.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Cumulocity IoT Edge on Kubernetes - Public Preview
+
+Cumulocity IoT Edge can now be deployed and managed on a single node Kubernetes cluster using Cumulocity IoT Edge Kubernetes Operator (Edge Operator).
+
+Edge on Kubernetes managed by the Edge Operator enables you with:
+
+- **Unified Management**: Manage Edge through an Edge Custom Resource (CR) in consistent with standard Kubernetes resources. This approach provides a unified and familiar interface for administrators and operators.
+- **Desired State Management**: Specify the desired state of Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
+- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:34]:
```suggestion
- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Cumulocity IoT Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- **Desired State Management**: Specify the desired state of Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
+- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 
+- **Automation**: Edge CR empowers you to automate various operational aspects of Edge like scaling, upgrades, and configuration.
+- **Distribution**: Distribute and replicate the Edge deployments using Edge CR. Simplifies the deployment and accelerates the adoption of Cumulocity IoT platform across diverse environments.
+
+For more information about how to install and manage Cumulocity IoT Edge on Kubernetes, see [Cumulocity IoT Edge Kubernetes Operator](https://cumulocity.com/guides//edge-k8s/k8-edge-introduction/).
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:38]:
```suggestion
For more information on how to install and manage Cumulocity IoT Edge on Kubernetes, see [Cumulocity IoT Edge Kubernetes Operator](https://cumulocity.com/guides//edge-k8s/k8-edge-introduction/).
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+{{< c8y-admon-info >}}
+Public Preview: Functionality is provided with LIMITED support. Customers can open support cases, file bugs and request feature enhancements, which will be taken care of by the Support Team.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:41]:
```suggestion
**Public Preview**: This functionality is only provided with limited support. Customers can open support cases, file bugs and request feature enhancements, which will then be taken care of by the Support Team.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+{{< /c8y-admon-info >}}
+
+Future releases of Edge on Kubernetes include:
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:44]:
```suggestion
Future releases of Cumulocity IoT Edge on Kubernetes include:
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+- Cumulocity IoT DataHub Integration
+- Messaging Service and the microservice-based data broker
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:46]:
```suggestion
- The messaging service and the microservice-based data broker
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
+---
+weight: 60
+title: Release 10.17.0
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.17, includes new features, improvements, fixes and known issues Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:7]:
```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+### Mongo DB update
+
+MongoDB has been updated to version 5.0.14.
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** Diasble the microservice hosting feature and enable it again. See, [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:21]:
```suggestion
|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** To Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+|CIE-4376|UI|**Issue:** When switching to a language other than English, translations for the security menu and security page are not available.
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:22]:
```suggestion
|CIE-4376|UI|**Issue:** When switching to a language other than English, the translations for the security menu and security page are not available.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+- **Distribution**: Distribute and replicate the Edge deployments using Edge CR. Simplifies the deployment and accelerates the adoption of Cumulocity IoT platform across diverse environments.
+
+For more information about how to install and manage Cumulocity IoT Edge on Kubernetes, see [Cumulocity IoT Edge Kubernetes Operator](https://cumulocity.com/guides//edge-k8s/k8-edge-introduction/).
+
+{{< c8y-admon-info >}}
+Public Preview: Functionality is provided with LIMITED support. Customers can open support cases, file bugs and request feature enhancements, which will be taken care of by the Support Team.
+{{< /c8y-admon-info >}}
+
+Future releases of Edge on Kubernetes include:
+- Cumulocity IoT DataHub Integration
```
Comment (MWindel) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:45]:
```suggestion
- The integration of Cumulocity IoT DataHub
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
+---
+weight: 60
+title: Release 10.17.0
+layout: redirect
+---
+
+The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:7]:
```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+Download the Cumulocity IoT Edge VM based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:9]:
```suggestion
Download the Cumulocity IoT Edge VM-based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+## Cumulocity IoT Edge Appliance VM
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:11]: We do not use heading level 2, please change all headings accordingly.

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+### Mongo DB update
+
+MongoDB has been updated to version 5.0.14.
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** To Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:21]:
```suggestion
|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/10.17.0/edge/configuration/#configuring-microservices).
```

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+|CIE-4376|UI|**Issue:** When switching to a language other than English, the translations for the security menu and security page are not available.
+
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-17-0/whatsnew-10-17-0/).
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:24]: There is a similar statement above linking to the youtube video. Perhaps link to both places above?

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md (hunk 15 above)
Comment (sharath-jk) [reply content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:11]: Done.

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md (hunk 17 above)
Comment (sharath-jk) [reply content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:24]: Added this link along with YouTube link.

File: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md [hunk 18]
```diff
@@ -0,0 +1,44 @@
+---
+weight: 60
+title: Release 10.17.0
+layout: redirect
+---
+
+The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922) and [Cumulocity IoT 10.17](/release-10-17-0/whatsnew-10-17-0/).
```
Comment (BeateRixen) [inline content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md:7]:
```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see the [What´s New] section (/release-10-17-0/whatsnew-10-17-0/) and our video [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

**Review by BeateRixen — APPROVED** [review]

**Review by bbyreddy — APPROVED** [review]

**Pull request merged**